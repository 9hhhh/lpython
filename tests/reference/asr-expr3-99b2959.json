{
    "basename": "asr-expr3-99b2959",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/expr3.f90",
    "infile_hash": "a517eaea31a1369bd05123b7a3553245b6a1e560bee961b85fef4a9a",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-expr3-99b2959.stdout",
    "stdout_hash": "c7992c63b0c67bb82fb7342c9ac3b8b80de2107934e757030c8b10ad",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}