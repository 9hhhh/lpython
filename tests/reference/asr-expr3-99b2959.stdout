(TranslationUnit (SymbolTable 00000003 {}) [(Num 5 (Integer 4 [])) (BinOp (Num 5 (Integer 4 [])) operatorType0 (Num 3 (Integer 4 [])) (Integer 4 [])) (BinOp (BinOp (Num 5 (Integer 4 [])) operatorType0 (Num 3 (Integer 4 [])) (Integer 4 [])) operatorType2 (Num 2 (Integer 4 [])) (Integer 4 [])) (BinOp (Num 5 (Integer 4 [])) operatorType0 (BinOp (Num 3 (Integer 4 [])) operatorType2 (Num 2 (Integer 4 [])) (Integer 4 [])) (Integer 4 [])) (BinOp (Num 5 (Integer 4 [])) operatorType1 (Num 3 (Integer 4 [])) (Integer 4 [])) (BinOp (Num 4 (Integer 4 [])) operatorType4 (Num 3 (Integer 4 [])) (Integer 4 []))])
